src/online2bin/online2-tcp-nnet3-decode-faster_word_timestamps.cc

```diff
@@ -355,35 +357,38 @@ int main(int argc, char *argv[]) {
                 last_timestamp = last_timestamp + offset;
                 str_end = std::to_string(last_timestamp);
                 current_hypothesis = current_hypothesis + "{\"word\":" + "\"" + word + "\"," + "\"start\":" + str_start + "," + "\"end\":" + str_end + "},";
-              }
-              if (!current_hypothesis.empty()) {
-                current_hypothesis.erase(std::prev(current_hypothesis.end()));
+                word_count += 1;
+                global_block_end = str_end;
               }
 
+              if (word_count > 0) {
+              current_hypothesis.erase(std::prev(current_hypothesis.end()));
               struct timeval tp;
               gettimeofday(&tp, NULL);
               long int timestamp = tp.tv_sec * 1000 + tp.tv_usec / 1000;
               std::string current_block = std::to_string(block);
               std::string block_identifier = std::to_string(timestamp);
-              current_hypothesis = "\"block_uuid\":\"" + block_uuid + "\"" + ", " +  "\"block\":" + current_block + ", " + "\"timestamp\": " + block_identifier + ", " + "\"words\":[" + current_hypothesis + "]}";
+              current_hypothesis = "\"block_uuid\":\"" + block_uuid + "\"" + ", " +  "\"block\":" + current_block + ", " + "\"timestamp\": " + block_identifier + ", " + ", \"block_word_start\": " + global_block_start + ", \"block_word_end\": " + global_block_end + ", " + "\"words\":[" + current_hypothesis + "]}";
```

Comment:
Could we maybe put these long lines in multiple code lines to make it more readable?